kadasrouting/gui/reachibilitybottombar.py

```diff
@@ -107,10 +110,8 @@ def createLayer(self, name):
 
     def calculate(self):
         clear = self.checkBoxRemovePrevious.isChecked()
```

Comment:
It seems this checkBoxRemovePrevious is not being used.

Since we have a custom plugin layer (`IsochronesLayer`) we can iterate all layers, and remove the one that is an `IsochronesLayer` object.

Reply:
But let's ask the user if they want to remove all previous isoline layer or only the very recent one. In the second case, we should store it somewhere.

Reply:
added a push warning to the user and added a dedicated `OverwriteError(Exception)` class for that